pkg/kube/longhorn-generate-support-bundle.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
```

Comment:
I think we should have a Copyright and License line here.

Reply:
Fixed